### PR TITLE
feat(node-wasm): Implement easier way for handling JS errors.

### PR DESCRIPTION
**Pull request by oblique**

### CODE REVIEW

File: node-wasm/src/error.rs [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+pub(crate) struct Error(JsValue);
+
+impl Error {
+    /// Create a new `Error` with the specified message.
+    pub(crate) fn new(msg: &str) -> Error {
+        Error(js_sys::Error::new(msg).into())
+    }
+
+    /// Create a new `Error` from `JsValue` without lossing any information.
```
Comment (fl0rek) [inline node-wasm/src/error.rs:17]:
```suggestion
    /// Create a new `Error` from `JsValue` without losing any information.
```

File: node-wasm/src/utils.rs [hunk 2]
```diff
@@ -71,27 +73,10 @@ pub(crate) fn js_value_from_display<D: fmt::Display>(value: D) -> JsValue {
...
-    where
-        C: fmt::Display + Send + Sync + 'static,
-    {
-        self.map_err(|e| JsError::new(&format!("{context}: {e}")))
-    }
-}
-
-pub(crate) fn get_crypto() -> Result<Crypto, JsError> {
+pub(crate) fn get_crypto() -> Result<Crypto, Error> {
     js_sys::Reflect::get(&js_sys::global(), &JsValue::from_str("crypto"))
```
Comment (fl0rek) [inline node-wasm/src/utils.rs:77]: by the looks of it, it should work both in normal js and in worker, nice one!

Comment (oblique) [reply node-wasm/src/utils.rs:77]: Yes. That is why I did it like this.

**Review by fl0rek — APPROVED** [review]
🎸 🎸

**Pull request merged**